build/Containerfile

```diff
@@ -1 +1,26 @@
-FROM registry.redhat.io/ubi8/ubi-minimal:latest
```

Comment:
ignorant GO question incoming, does ubi-minimal not have the correct go version?

Reply:
It doesn't have any go version. Building from the go toolset and then moving the executable over saves a ton of space. We don't need all the build tools to exist on the running image